package.json

```diff
@@ -35,13 +35,13 @@
     "@backbase/arrangement-manager-http-ang": "^1.3.7",
     "@backbase/backbase-theme": "^2.316.0",
     "@backbase/data-ang": "^3.10.6",
-    "@backbase/foundation-ang": "^6.23.3",
+    "@backbase/foundation-ang": "^7.0.0-beta.1",
     "@backbase/positive-pay-check-v1-client-ang": "^1.3.0",
-    "@backbase/positive-pay-journey-ang": "^0.44.0",
-    "@backbase/select-context-widget-ang": "^4.4.0",
-    "@backbase/ui-ang": "^7.5.0",
-    "@backbase/universal-ang": "^7.0.0",
-    "@ng-bootstrap/ng-bootstrap": "^10.0.0",
+    "@backbase/positive-pay-journey-ang": "^2.0.0",
+    "@backbase/select-context-widget-ang": "^5.6.2",
+    "@backbase/ui-ang": "file:../web-ui/dist/libs/ui-ang/backbase-ui-ang-8.0.0-pr.37.tgz",
+    "@backbase/universal-ang": "^8.0.0",
```

Comment:
this should be removed with ui-ang 8 ?

Reply:
You are right. Removed it. But to make it work with ui-ang 8, I've installed these deps:

- "@ng-select/ng-select": "^8.1.1",
- "bignumber.js": "^9.0.1",
- "ngx-mask": "^13.0.0",